### PR TITLE
Feature: add env PATH into hijack sudo function

### PR DESCRIPTION
**Pull request by Bin-QA**
Now skip type 'sudo env PATH="PATH"' from case command

Signed-off-by: Wu, BinX <binx.wu@intel.com>

### CODE REVIEW

File: case-lib/hijack.sh [hunk 1]
```diff
@@ -76,11 +76,11 @@ sudo()
             return $?
         ;;
         '1')    # sudo without passwd
-            eval $(echo "$SUDO_CMD $*")
+            eval $(echo "$SUDO_CMD env 'PATH=$PATH' $*")
             return $?
         ;;
         '2')    # sudo need passwd
-            eval $(echo "echo '$SUDO_PASSWD' | $SUDO_CMD -S $*")
+            eval $(echo "echo '$SUDO_PASSWD' | $SUDO_CMD -S env 'PATH=$PATH' $*")
```
Comment (xiulipan) [inline case-lib/hijack.sh]: will `-E` help here to pass the ENV into sudo

Comment (Bin-QA) [reply case-lib/hijack.sh]: try with -E option which is not work in our script, because we are not `export` the ENV

Comment (Bin-QA) [reply case-lib/hijack.sh]: try with option `--preserve-env=PATH` it will report `command not found`

it seems is hijack effect

**Comment by xiulipan** [comment]
@aiChaoSONG @libinyang @keqiaozhang Please help to review.

**Comment by aiChaoSONG** [comment]
When I submit patches to kernel, Pierre told me I should explain my patch clearly in commit message. It's a what, how and why steps. you should make it clear that what the problems is(why it need a fix),  how you solve it, and why you solve it this way (safe? clarity? or ?). 
I suggest to make our commit message more verbose, what do you think? @xiulipan @Bin-QA 

@Bin-QA I see what you did in this patch, but I didn't get the reason. could you please explain deeper?

**Pull request merged**